### ISSUE TITLE
Strange performance on Microsoft Edge

### ISSUE THREAD

**Issue by bryo5363** [phetsims/friction#31]
Test device: Acer Aspire R13 (Bryan's personal computer)

Operating System: Windows 10 64 bit

Browser: Microsoft Edge 20.10240.16384.0

Problem description: The sim does not function properly on this browser. The temperature bar will turn black when contact is made and the particle alignment is compromised when resetting the sim

Steps to reproduce:
1) open sim
2) drag book. the temperature bar will we black rather than red.
3) drag book until the base level of particles are left on the yellow book. 
4) reset the sim. Notice that the base level will be much higher than initially set

Severity:

Screenshots:
<img width="599" alt="friction1" src="https://cloud.githubusercontent.com/assets/7717368/10642830/fe3a234a-77dc-11e5-9fc0-83e3e6d59cba.png">

<img width="596" alt="picture2" src="https://cloud.githubusercontent.com/assets/7717368/10642889/331bba24-77dd-11e5-9dcb-3710f460a247.png">

Troubleshooting information (do not edit):

Name: ‪Friction‬

URL: http://www.colorado.edu/physics/phet/dev/html/friction/1.2.0-rc.1/friction_en.html

Version: 1.2.0-rc.1 2015-10-20 23:08:29 UTC

Features missing: touch

Flags: pointerEnabled, pixelRatioScaling

User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10240

Language: en-US

Window: 800x554

Pixel Ratio: 1.5/1

WebGL: WebGL 0.95

GLSL: WebGL GLSL ES 0.95

Vendor: Microsoft (Internet Explorer)

Vertex: attribs: 16 varying: 15 uniform: 512

Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)

Max viewport: 16384x16384

OES_texture_float: true

Dependencies JSON: {"assert":{"sha":"11bc6a6d","branch":"HEAD"},"axon":{"sha":"e2237347","branch":"HEAD"},"babel":{"sha":"931afbf1","branch":"master"},"brand":{"sha":"8a756e53","branch":"HEAD"},"chipper":{"sha":"157b9cf3","branch":"HEAD"},"dot":{"sha":"d99cbe46","branch":"HEAD"},"friction":{"sha":"7b7bd70f","branch":"1.2"},"joist":{"sha":"121bb0de","branch":"HEAD"},"kite":{"sha":"40dc703a","branch":"HEAD"},"phet-core":{"sha":"e9f36af1","branch":"HEAD"},"phetcommon":{"sha":"0e4754fe","branch":"HEAD"},"scenery":{"sha":"c1d6232d","branch":"HEAD"},"scenery-phet":{"sha":"88ade507","branch":"HEAD"},"sherpa":{"sha":"1759eade","branch":"HEAD"},"sun":{"sha":"b66398a5","branch":"HEAD"},"tandem":{"sha":"c54f1082","branch":"HEAD"}}

**Comment by ariel-phet** [comment]
@jbphet - this seems odd...

**Comment by ariel-phet** [comment]
@jonathanolson - you might want to take a peek at this issue as well

**Comment by jbphet** [comment]
I am able to duplicate on the Dell Studio XPS that was upgraded to Windows 10.  I've done a couple of experiments, and it appears that the problem is related to the clip area.  Removing the clip area entirely makes the problem go away, but it also makes it so that the liquid in the thermometer keeps going well above the top of the thermometer.  We could probably make a workaround that only uses shape and doesn't rely on clip area, but I'd rather not do that if we don't have to.

**Comment by jbphet** [comment]
I'm also able to duplicate the reset issue on the Dell Studio XPS Win 10 running Edge.

**Comment by jbphet** [comment]
I contacted @jonathanolson about these issues, and he and I looked into them.  These issues only show up when using canvas as a renderer, which we are currently always doing for Edge due to some other issues.  The same odd behavior shows up in other browsers when the renderer is forced to be canvas.  @jonathanolson dug into it and fixed the clipping issue with this commit: https://github.com/phetsims/scenery/commit/2edf8a0154d7be46b34b973483459df1c7f9782d.  He fixed the reset behavior with this commit: https://github.com/phetsims/scenery/commit/aacceb9a2835e1803c4b1c88bd8c112e70a16314.  Testing of the requireJS version using the Edge browser seems to indicate that the problems are resolved.  I'll update the version of Scenery that is used on this RC branch and roll a new release candidate.
